app/models/miq_report/seeding.rb

```diff
@@ -84,12 +86,31 @@ def seed_record(path, report)
       end
     end
 
+    def template_type(path)
+      path.include?(RELATIVE_REPORT_DIR.to_s) ? "report" : "compare"
+    end
+
     def seed_files
+      seed_core_files + seed_plugin_files
+    end
+
+    def seed_core_files
       Dir.glob(REPORT_DIR.join("**/*.yaml")).sort + Dir.glob(COMPARE_DIR.join("**/*.yaml")).sort
     end
 
+    def seed_plugin_files
+      Vmdb::Plugins.flat_map do |plugin|
+        Dir.glob(plugin.root.join("#{RELATIVE_REPORT_DIR}/**/*.yaml")).sort + Dir.glob(plugin.root.join("#{RELATIVE_COMPARE_DIR}/**/*.yaml")).sort
+      end
+    end
+
     def seed_filename(path)
-      path.remove("#{REPORT_DIR}/").remove("#{COMPARE_DIR}/")
+      case template_type(path)
+      when "report"
+        path.split("#{RELATIVE_REPORT_DIR}/").last
+      when "compare"
+        path.split("#{RELATIVE_COMPARE_DIR}/").last
```

Comment:
This was the best I could come up to handle these two cases, `File.basename(path)` works for the "content" type which looks like `product/compare/ems_clusters.yaml` but not for "report" which looks like `product/reports/420_Operations - Clusters/010_Cluster - DRS migrations.yaml` and by the same token `path.split("/")[-2..-1].join("/")` won't work for the "compare" type

Reply:
Okay that's a little better

Reply:
Aaaand I'm talking to myself

Reply:
First step is acknowledgement....